src/Checkpointer.jl

```diff
@@ -160,7 +160,9 @@ the component models that have a cache.
 function restart_model_cache!(sim, input_file)
     ispath(input_file) || error("File $(input_file) not found")
     # Component models are responsible for defining a method for this
-    restore_cache!(sim, JLD2.jldopen(input_file)["cache"])
+    file = JLD2.jldopen(input_file)
+    restore_cache!(sim, file["cache"])
+    close(file)
```

Comment:
```suggestion
    JLD2.jldopen(input_file) do file
        restore_cache!(sim, file["cache"])
    end
```
I kind of prefer using the `do`-syntax, as this will gracefully handle errors, and it's a nice pattern to repeat (in this case, the file should be closed by `JLD2`)

Reply:
Thanks, I was not aware of this! 👍